### PR TITLE
Bookpage

### PR DESCRIPTION
**Pull request by Satishchandra123**

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -90,4 +91,21 @@ def authenticate():
...
+# @app.route("/bookpage/<string:isbn_id>")
+# def book_details(isbn_id):
+#     book = db1.session.query(Books).filter(Books.isbn == isbn_id).all()
+#     return render_template("bookpage.html", data=book)
+
+@app.route("/bookpage/<string:isbn>")
+def books(isbn):
+    # ISBN = Books.query.get("isbn")
+    isbn = "1416949658"
+    details =  Books.query.filter_by(isbn = ISBN)
```
Comment (rehana-ms) [inline project1/application.py]: You will get  only exact results.Try to  work on partial results as well.For example,If I enter 141 for isbn,it should display all the books starting with that series.

File: project1/templates/bookpage.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <td>Author</td>
+                <td>{{book.author}}</td>
+            </tr>
+            <tr> 
+                <td>Published</td>
+                <td>{{book.year}}</td>
+            </tr>
+            {% endfor %}      
+            </tbody>
+            {% for item in result %}
```
Comment (rehana-ms) [inline project1/templates/bookpage.html]: from where are you getting result values?

Comment (rehana-ms) [reply project1/templates/bookpage.html]: Can you explain.

File: project1/application.py (hunk 1 above)
Comment (Satishchandra123) [reply project1/application.py]: Mam, we have divided our work into 3 parts. search, bookpage and Review. That displaying of books based on the search is being done by priyanka.  My work is that If I had clicked that particular isbn I should display all the details of that particular book.


Capture

File: project1/templates/bookpage.html (hunk 2 above)
Comment (Satishchandra123) [reply project1/templates/bookpage.html]: Sorry mam actually its not the part of code. I need to comment them but I forgot. The below mentioned is the actual thing from where I have received the data and printed in the table.  The result thing is different which I worked on further but forgot to comment it.




fg

Comment (rehana-ms) [reply project1/templates/bookpage.html]: ok

File: project1/application.py (hunk 1 above)
Comment (rehana-ms) [reply project1/application.py]: ok

**Comment by rehana-ms** [comment]
good

**Pull request merged**